src/ncdiag/cmake/ncdiag_compiler_flags_Intel_Fortran.cmake

```diff
@@ -2,7 +2,7 @@
 # FLAGS COMMON TO ALL BUILD TYPES
 ####################################################################
 
-set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -assume byterecl -convert big_endian -implicitnone")
+set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -g -traceback -assume byterecl -convert big_endian -implicitnone")
```

Comment:
Can you please update the `ncdiag_compiler_flags_GNU_Fortran.cmake` to include `-g -fbacktrace`?

Reply:
Interestingly, both -g and -fbacktrace were already present in ncdiag_compiler_flags_GNU_Fortran.cmake.